README.md

```diff
@@ -1,2 +1,243 @@
+* [Git Essential Training](https://www.lynda.com/Git-tutorials/Git-Essential-Training/100222-2.html),
+an on-line Lynda.com course; it's free if you first authenticate
+with your U of M credentials via http://lynda.umn.edu
+
+The discussion below assumes that people are paired up in the lab, and
+there will be times where one person will need to log out and let their
+partner login so that everyone has a chance to go through all the
+activities.
+
+:bangbang: This repository will be private so only people in the class
+(and the faculty and TAs) will have access to this contact info.
+Everyone in the class will have access to it, though, so you should
+put some thought in what you want to share.
+
+# Create a contact info page for (each of) you
+
+This repository has a `contact_info` directory, and in this part of the
+lab we'll create a new file in that directory for every person in the
+class, containing their personal contact info.
+
+:question: KK – do we want to have person X type up person Y's info
+and vice versa so everyone is in some way involved all through the
+process? I'm going to write it up that way below, but we can definitely
+change that if you think that's a bad idea.
+
+In order to keep everyone engaged throughout, we'll model the ideas of
+a _driver_ and _navigator_ in [pair programming](https://en.wikipedia.org/wiki/Pair_programming).
+Imagine, for example, a group of two working together, Pat and Chris,
+and Pat is currently logged in. Then Pat would create the contact info
+document for _Chris_, asking Chris for the relevant information and
+entering it. When that's done, Pat will commit that work, and then log
+out, allowing Chris to log in and repeat the process creating _Pat's_
+contact info page.
+
+## Creating a branch
+
+`git` branches are frequently used as ways to isolate work (in progress)
+on a particular feature from other on-going work on the project.
+Creating and working in your own feature branch allows you to commit
+and share incomplete work without breaking the project for other people.
+
+To illustrate the use of branches (which arguably aren't really needed
+for this simple project), we'll create a new branch for the creation
+of each contact info page. So in the example above, the first thing
+that Pat would do would be to create a branch for the creation of
+Chris's contact info page.
+
+In the command line this would be accomplished with
+```bash
+git checkout -b <branch_name>
+```
+so Pat might use `git checkout -b chris_info` to create the branch for
+setting up Chris's info.
+
+In WebStorm, you can accomplish this through the
+`VCS -> Git -> Branches…` dialogue, choosing the `New Branch` option.
+
+## Creating a page
+
+Once you've created the branch it's time to create your partner's
+contact info page.
+
+Each contact info page should:
+
+* Be named `<your_name>.md`, e.g., `Nic_McPhee.md`
```

Comment:
It won't be "your" name, but it will be the partner's name. It's the name of the person whose contact info will be there.